packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+* Alice: (NAT IP + portal123 + Device1 ID) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* Bob: (NAT IP + portal123 + Device2 ID) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* Lisa: (NAT IP + portal123 + Device3 ID) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+
+> **Note:** The device identifier is derived from a cookie (`dt` cookie) that Okta sets in the browser when the first request is made to Okta. When the requests are made using a non-browser client, the device cookie isn't present. In that case, such requests fall under a common quota with the device identifier being null (NAT IP + portal123 + null).
+
+#### Client-based isolation for users accessing `/authorize` through a proxy
+
+When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limiting framework to look for the IP address before the proxy to find the true client IP address.
+
+### Client-based rate-limiting modes
+
+The client-based rate-limiting framework can exist in one of three modes:
+
+| Mode                                     | Description                                                                                                          |
+| ---------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| **Enforce and log per client (recommended)** | Rate limiting is based on client-based rate-limiting values. Client-specific rate limiting violation information is logged by firing System Log events. |
+| **Log per client**                          | Rate limiting is still based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values, but client-specific rate limiting violation information is logged by firing System Log events. |
+| **No action**                                | Rate limits aren't enforced at the client-specific level. Rate limiting is based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values. |
+
+### System Log events
+
+When an individual user violates the per minute requests limit assigned to them, depending on which mode is set, the following System Log event is fired:
+
+* `system.client.rate_limit.violation`: This event is fired when the framework is in **Enforce and log per client** mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
```

Comment:
The 60 requests per minute limit isn't mentioned in the rate limits file (https://github.com/okta/okta-developer-docs/pull/1467/files#diff-b824ac7f01672ca523e3053b27302b62), only here. If it's a hard limit, shouldn't it be in that file as well?

(Shorten by removing the repetitive phrase):
This event is fired when the framework is in **Enforce and log per client** mode **and** a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. If the user already has a valid session, then the System Log contains information about the client ID, IP address, device identifier, and the actual user.

Reply:
was trying to avoid the overly long sentence. but, it just works better really long.